ui/src/components/SampleQueue/CurrentTree.jsx

```diff
@@ -147,14 +120,8 @@ function CurrentTree(props) {
                 <CharacterisationTaskItem
                   key={taskData.queueID}
                   index={i}
-                  id={`${taskData.queueID}`}
                   data={taskData}
                   sampleId={sampleId}
-                  checked={checked}
-                  showForm={(...args) => dispatch(showTaskForm(...args))}
-                  addTask={addTask}
```

Comment:
The `addTask` action was not being dispatched. This is a regression I introduced last week in #1984. It is now fixed (cf. `CharacterisationTaskItem`)